.env.sample

```diff
@@ -0,0 +1,9 @@
+SECRET_KEY=nad7kycy!op3$^p@37=xk!mcwmmv5zw6@*e$r+#m5ood2)@i78
+DEBUG=on
```

Comment:
Needs to be either True or False right?

Reply:
Works either ways, but I'll change it to True/False